src/collection-view.js

```diff
@@ -482,7 +487,13 @@ Marionette.CollectionView = Marionette.View.extend({
 
   // render the child view
   renderChildView: function(view, index) {
+    if (!(view instanceof Marionette.View)) {
```

Comment:
Oh, I get it. We're manually firing the event on B.V ha. okay, I like this. sorry took me awhile